sections/plate_notation.tex

```diff
@@ -84,8 +84,27 @@ \subsection{Pachinko Allocation}
 This \gls{dag} structure is visualized in \autoref{fig:pachinko_dag}.
 
 For each word, a chain of topics is sampled by sampling one topic from each layer of the structure, and finally, a word is chosen based on the last topic.
-More specifically, for each topic starting with the root topic, we sample a topic from the next layer $Z_{w(l+1)}$ from that node's topic distribution $\theta_l^{(d)}$ based on observed data from that document.
+More specifically, for each topic starting with the root topic $Z_{w1}$, we sample a topic from the next layer $Z_{w(l+1)}$ from that node's topic distribution $\theta_l^{(d)}$ based on observed data from that document.
+In \autoref{eq:pachinko_gibbs}, the joint probability for our model is shown.
+This calculation corresponds to the generative process in \autoref{fig:pachinko}, where each level of the Pachinko is sampled in the inner plate.
+
+\begin{equation}\label{eq:pachinko_gibbs}
+	\begin{split}
+		P(Z_{w2} = t_a, Z_{w3} = t_b, Z_{w4} = t_c | \textbf{D}, z_{-w}, \alpha, \beta) \propto \\
+		\frac{n_{1a}^d + \alpha_{1a}}{n_1^d + \sum_{a'} \alpha_{1a'}} \times
+		\frac{n_{ab}^d + \alpha_{ab}}{n_r^d + \sum_{b'} \alpha_{ab'}}  \times 
+		\frac{n_{bc}^d + \alpha_{bc}}{n_{k}^d + \sum_{c'} \alpha_{bc'}} \\ \times 
+		\frac{n_{cw}^d + \beta_{w}}{n_{c} + \sum_{m} \beta_{m}} 
+	\end{split}
+\end{equation}
+As in \citet{li2006pachinko} $t_1$ is the root topic and $Z_{w2}$, $Z_{w3}$, $Z_{w4}$ are the three middle layers of topics in our 5-layer Pachinko model.
+$Z_{-w}$ is the word topic assignment, for all other words than the one we are working on.
+Without looking at the current object of iteration, $n_x^d$ is the number of times topic $tx$ occurs in document $d$. 
+The $n_{xy}^d$ describes how many times topic $t_y$ is sampled from its parent $t_x$ within document $d$.
+$n_x$ is the number of times subtopic $t_x$ occurs in the corpus and $n_{xw}$ is the number of times a word $w$ is in $t_x$.
+
 The plate notation for the \gls{pam} can be seen in \autoref{fig:pachinko}.
+The idea behind using five layers instead of four is because, the taxonomy field contains up to 5 layers of information. 
```

Comment:
```suggestion
\todo{describe why we use 5 layers.}
```
this is not true. its more complex than that. Vi bruger kun 2 lag fra taxonomy ikke 5.